src/types/index.ts

```diff
@@ -376,6 +381,37 @@ export interface TextFileOnChangeEvent {
   latestContent: string;
 }
 
+export enum ChangeEventType {
+  Create = "CREATE",
+  Move = "MOVE",
+  Delete = "DELETE",
+  Modify = "MODIFY",
+}
+export interface FsMoveEvent {
+  node: FsNode;
+  to: string;
+  eventType: ChangeEventType.Move;
+}
+
+export interface FsDeleteEvent {
+  node: FsNode;
+  eventType: ChangeEventType.Delete;
+}
```

Comment:
There's already 2 events in this same file. We gotta split this into multiple files, ideally colocate the types with the modules soon, this getting painful

Reply:
good idea to change it to an enum

Reply:
I'll split types into different files as a followup PR, after this one is merged 👍🏽

Reply:
Will make the enum type for now